src/main/java/com/spotinst/sdkjava/model/DynamicVolumeSize.java

```diff
@@ -0,0 +1,117 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by sobhana p on 8/13/15.
+ */
+public class DynamicVolumeSize {
+
+    //region Members
+    @JsonIgnore
+    private Set<String>              isSet;
+    private Integer                  baseSize;
+    private Integer                  sizePerResourceUnit;
+    private String                   resource;
```

Comment:
please cross-check with postman once, this is ENUM

Reply:
<img width="1011" alt="image" src="https://github.com/spotinst/spotinst-sdk-java/assets/86217539/19a22ee4-4471-47d4-aaf3-37dbc801e024">
Not Enum String only